style.css

```diff
@@ -0,0 +1,419 @@
+* {
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+body {
+    padding: 0;
+    margin: 0;
+    font-size: 62.5%;
+    background-image: url("images/bubble-background-4.jpg");
+    background-size: cover;
+}
+
+/* Splash Screen Styling */
+
+#splash-screen {
+    position: relative;
+    width: 80%;
+    margin: 0 auto;
+}
+
+.fish-logo {
+    width: 12%;
+    position: absolute;
+    top: 5%;
+    left: 10%;
+    animation: makeFishMove 1s linear 0s infinite alternate;
+}
+
+/* Fish Logo Animation */
+
+@keyframes makeFishMove {
+    from {transform: rotate(5deg);}
+    to{transform: rotate(-5deg);}
+}
+
+.splash-container {
+    position: relative;
+    margin: 0 auto;
+    text-align: center;
+    width: 61%;
+}
+
+h1 {
+    font-family: 'Ranchers', cursive;
+    font-size: 4rem;
+    letter-spacing: 0.2rem;
+    color: #e4e726;
+    margin-bottom: 0px;
```

Comment:
don't need a unit of measure, just 0 is fine. there's a couple other instances of this too.